main/res/values/strings.xml

```diff
@@ -683,6 +683,13 @@
         <item quantity="other">%1$d caches</item>
     </plurals>
 
+    <!-- days remaining -->
+    <plurals name="days_remaining">
+        <item quantity="one">one day remaining</item>
+        <item quantity="other">%1$d days remaining</item>
+    </plurals>
+    <string name="last_day_available">last day available</string>
```

Comment:
I had zero in the first place, but zero isn't recognized for some languages like english, because they say it's the same as plural.
See: https://stackoverflow.com/questions/17261290/plural-definition-is-ignored-for-zero-quantity